codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
+  - "/home/runner/work/iaux-typescript-wc-template/iaux-typescript-wc-template/::" # Correct paths
```

Comment:
What is this file configuring?

Reply:
It is configuring the directory where the tests are running to be used by the bash later.

Reply:
This file might not be needed if you use the `npx codecov` tool